CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+##  Next version
+
+This patch release contains the following fix:
+- This Patch introduces Filter Protocol timeout.
+- Support for full secure websockets.
```

Comment:
```suggestion
This release contains the following:
```

A good idea is to take a look at one of the past versions of CHANGELOG before a release, e.g. [here](https://github.com/status-im/nim-waku/blob/d1e06fa17ab7989e487b095548543df0936399fb/CHANGELOG.md), to see how the `Next version` section is structured.

The specifics about `filter` protocol and `websockets` should be moved to a subheading below.